synapse/models/person.py

```diff
@@ -43,6 +45,8 @@ def getModelDefs(self):
 
                 ('ps:contact', ('guid', {}), {
                     'doc': 'A GUID for a contact info record.',
+                    'interfaces': ('phys:object',),
```

Comment:
I am iffy on having ps:contact inherit phys:object:
- a contact could be for an org (probably would not  include height/weight/mass of "headquarters") as well as a person
- we have ps:vitals, which  can be linked to a ps:contact / ps:person / both.
So it seems the only advantage to putting height/weight/mass on the contact is to have it "one degree closer", but this potentially splits tracking of this data on to two forms (the ps:contact and related ps:vitals...three if you include the "most current" (?) stats on ps:person).